src/rgw/rgw_main.cc

```diff
@@ -1276,6 +1282,7 @@ int main(int argc, const char **argv)
   curl_global_cleanup();
 
   rgw_perf_stop(g_ceph_context);
+  percentile_perf_stop(g_ceph_context);
```

Comment:
--admin-daemon * perf dump gives a set of collection of performance counters
"client.rgw.stage-d42sa-rgw-a-0003-416610": {
        "http_1xx_count": 167,
        "http_2xx_count": 2999,
        "http_3xx_count": 0,
        "http_4xx_count": 137,
        "http_5xx_count": 0,
        "http_NULL_count": 0
},
"objecter": {
},
"percentiles": {
        "Put_ObjectSize_65536_Percentile_0.7": 14,
        "Put_ObjectSize_65536_Percentile_0.9": 96,
        "Put_ObjectSize_65536_Percentile_0.99": 312,
        "Put_ObjectSize_65536_Percentile_0.999": 312,
}, "throttle-objecter_ops": {
}


rgw_perf_start and rgw_perf_stop deals with only "client.rgw.stage-d42sa-rgw-a-0003-416610".